src/layouts/DashboardLayout/TopBar.tsx

```diff
@@ -11,13 +11,25 @@ import {
   Toolbar,
   makeStyles
 } from '@material-ui/core';
+import FormControl from "@material-ui/core/FormControl";
+import InputIcon from '@material-ui/icons/Input';
+import InputLabel from "@material-ui/core/InputLabel";
 import MenuIcon from '@material-ui/icons/Menu';
+import MenuItem from "@material-ui/core/MenuItem";
 import NotificationsIcon from '@material-ui/icons/NotificationsOutlined';
-import InputIcon from '@material-ui/icons/Input';
+import Select from "@material-ui/core/Select";
 import Logo from 'src/components/Logo';
 
 const useStyles = makeStyles(() => ({
   root: {},
+  formControl: {
```

Comment:
what if we use yet another formControl? (think about the naming and reusability...)